ex18_praticepython.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Dec  5 12:08:47 2017
+
+@author: Anna
+"""
+
+#ex18
+
+import random
+
+ra=[]
+for i in range(4):
+    i=random.randint(0,9)
+    ra.append(i)
+
+stra=''.join(str(e) for e in ra)
+zgadnij=list(input("Podaj liczbe czterocyfrowa: "))
+nowa=[]
+for item in zgadnij:
+    nowa.append(int(item))
+
+def porownanie(ra,nowa):
+    krowabyk=[0,0]
+    for i,ra_val in enumerate(ra):
+        #print (i,ra_val)
+        for j,nowa_val in enumerate(nowa):
+            if ra_val==nowa_val and i==j:
+                krowabyk[0]+=1
+            elif ra_val==nowa_val and i!=j:
+                krowabyk[1]+=1
+            
+    return krowabyk            
+
+wynik=porownanie(ra,nowa)
```

Comment:
- Nie spełniony warunek: Until the user guesses the number.
- Brakuje funkcji main
- Wynik nie pozwala na łatwe zgadnięcie kodu

```
python3 ex18_praticepython.py 
Podaj liczbe czterocyfrowa: 1111
[1, 3]
[6, 1, 6, 8]
```

Liczba oznaczona jako krowa nie powinna już być uwzględniana jako byk. Zajęty byk nie powinien tez już generować kolejnych byków. Nie wiem czy to jasne ale oczekiwałbym następującego wyniku:
- losowa liczba: 1234
- liczba użytkownika: 1022
- wynik: 1 krowa + 1 byk